src/js/controllers/chatbot-widget.js

```diff
@@ -96,10 +97,15 @@
                 $scope.$on('chatbotMessageReplyClicked', vm.sendReply);
 
                 vm.addToChat = function (message) {
-                    var newData = [].concat(vm.data, [
-                        Object.assign({}, message)
-                    ]);
-                    this.data = newData;
+                    if (message.type === "text" && message.message === "") {
```

Comment:
Zelfde opmerking als hierboven. Het is nogal vreemd om een lege conditional te definiëren.

Reply:
Ik vond het belangrijk om expliciet te documenteren waarom de if conditie geen gevolgen heeft, maar ik had de comment ook voor de if kunnen plaatsen.